package.json

```diff
@@ -18,18 +18,20 @@
   ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/Microsoft/clarity-tools.git"
+    "url": "git+https://github.com/Microsoft/clarity-tools.git"
   },
   "bugs": {
     "url": "https://github.com/Microsoft/clarity-tools/issues"
   },
   "dependencies": {
+    "clarity": "^1.0.2",
```

Comment:
> "clarity": "^1.0.2", [](start = 3, length = 21)

why was this added? 

[https://www.npmjs.com/package/clarity](https://www.npmjs.com/package/clarity)

That's not our package